uaa/slateCustomizations/source/index.html.md.erb

```diff
@@ -871,6 +871,7 @@ _Error Codes_
 | 401        | Unauthorized - Invalid token                                                |
 | 403        | Forbidden - Insufficient scope (zone admins can only delete their own zone) |
 | 404        | Not Found - Zone does not exist                                             |
+| 422        | Unprocessable Entity - at least one IdP with alias exists in the zone       |
```

Comment:
Unprocessable Entity (e.g., deleting IdP with alias while aliasEntitiesEnabled is true)

Found similar fragments in https://docs.cloudfoundry.org/api/uaa/version/77.8.0/index.html#delete

Reply:
see my comment above - here, the flag is not relevant